test/object.civet

```diff
@@ -794,6 +794,22 @@ describe "object", ->
       ({z: x.y.z})
     """
 
+    testCase """
+      empty
+      ---
+      {{}}
+      ---
+      ({})
+    """
+
+    testCase """
+      empty
+      ---
+      {{}, {}}
+      ---
+      ({})
+    """
```

Comment:
Not sure I understand... Is this being treated like `{...{}, ...{}}`? Not really sure why nested object literals should be allowed.

Reply:
I believe it's being treated like `a.{x, y}` but with both parts empty.